src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -18,19 +21,33 @@ import TableHeader from "components/molecules/TableHeader";
 import { UserList } from "components/molecules/UserList";
 import { TablesUserList } from "components/molecules/TablesUserList";
 
+import { BackButton } from "components/atoms/BackButton";
+
 import { TABLES } from "./constants";
 
 import "./ConversationSpace.scss";
 
 // @debt refactor this to pass in venue as a prop
 export const ConversationSpace: React.FunctionComponent = () => {
   const venue = useSelector(currentVenueSelectorData);
+  const venueId = useVenueId();
+
```

Comment:
could you please just pass the venue from the VenueTemplate and remove these?

Reply:
fixed in 0c10dd0ff4e60a817be81910ee3c365705a20cb6